src/controller/java/JniReferences.h

```diff
@@ -17,17 +17,29 @@
 
 #pragma once
 
+#include <core/CHIPError.h>
 #include <jni.h>
+#include <pthread.h>
+#include <support/CodeUtils.h>
 
-#include <core/CHIPError.h>
+class JniReferences
```

Comment:
Everything in this class seems to be static. Would a namespace be more appropriate here?

Alternatively, this should be a singleton class with some ::Instance method so that not all methods are static.

Reply:
#7802 7802